ducktape/command_line/config.py

```diff
@@ -43,3 +43,7 @@ class ConsoleConfig(object):
     # Log this to indicate a test is misbehaving to help end user find which test is at fault
     BAD_TEST_MESSAGE = "BAD_TEST"
 
+    # ducktape version
+    DUCKTAPE_VERSION = "0.2.0"
```

Comment:
Kind of weird for this to be under the command line part since it's just the version of the entire package.

The normal place to put this is in the packages __init__.py. See, e.g., https://github.com/kennethreitz/requests/blob/master/requests/__init__.py#L44 for an example (which also includes other metadata). They also go through pains to avoid duplicating it so there's no chance of things getting out of sync: https://github.com/kennethreitz/requests/blob/master/setup.py#L31 Might want to follow a similar approach.